.travis.yml

```diff
@@ -9,14 +9,18 @@ matrix:
       osx_image: xcode9.4
       xcode_scheme: ReactiveTask
       script:
-        - set -o pipefail && xcodebuild -workspace "$TRAVIS_XCODE_WORKSPACE" -scheme "$TRAVIS_XCODE_SCHEME" test | xcpretty
+        - set -o pipefail && xcodebuild -workspace "$TRAVIS_XCODE_WORKSPACE" -scheme "$TRAVIS_XCODE_SCHEME" test | bundle exec xcpretty
       env: JOB=Xcode9.4
     - os: osx
       osx_image: xcode9.4
+      before_script:
+        - bundle install
       script:
         - swift --version
         - swift build
         - swift test
+        - bundle exec pod repo update
+        - bundle exec pod spec lint --quick
```

Comment:
This should be a separate build phase. There's no reason to tie it to the SWIFTPM_DARWIN build.